Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/css/search.css

```diff
@@ -45,4 +45,18 @@
 .search-candidates .selected {
     background-color: var(--blue);
     color: white;
+}
+
+.next-regress-bar {
```

Comment:
Can we add some padding on top here?

Reply:
Agree on the padding, the top of the button ends up hidden when scrolling at the moment.